metabase/metabase
# Make custom setup buttons accessible

Epic: https://github.com/metabase/metabase/issues/19696

Fixes keyboard a11y issues with some buttons in setup:
- `Remove selected database` button - shown when you select a database
- `Show advanced options` button - shown when you select a database
- `I'll add my data later` link - on the database step
- `Database help` - the card shown on the right when you select a database

How to test:
- Make sure all these buttons/links receive keyboard focus
- It should be possible to interact with them via keyboard, e.g. by pressing `Enter`
